src/server/worldserver/worldserver.conf.dist

```diff
@@ -3361,3 +3361,32 @@ PacketSpoof.BanDuration = 86400
 
 #
 ###################################################################################################
+
+###################################################################################################
+# ARENA DYNAMIC MATCHMAKING SETTINGS
+#
+#    ArenaDynamicMatchmakingSystem.enable
```

Comment:
enable => Enable (capital first letter in keywords)
update => Update (capital first letter in keywords)